Source/Meadow.Foundation.Libraries_and_Frameworks/Displays.TextDisplayMenu/Samples/TextDisplayMenu_GameMenu_Sample/MeadowApp.cs

```diff
@@ -148,8 +148,10 @@ await Task.Run(() =>
                     microGraphics.DrawText(0, 0, $"{command}:");
                     microGraphics.DrawText(0, 20, $"{count++}");
                     microGraphics.DrawPixel(x += xD, y += yD);
-                    if (x == microGraphics.Width || x == 0) { xD *= -1; };
-                    if (y == microGraphics.Height || y == 0) { yD *= -1; };
+                    if (x == microGraphics.Width || x == 0) { xD *= -1; }
+                    ;
```

Comment:
wuuut???

Reply:
unneeded semicolons so the formatter put em on their own line - deleted